### PR TITLE
add subsection about handling projections

### PR DESCRIPTION
**Pull request by mradamcox**
Adding section 2.2.1 about projections, based on discussion in Slack.

I'm not sure if this is the best location for this information, but wanted to get something in PR form for you to look at.

### CODE REVIEW

File: source/extension/georef/index.md [hunk 1]
```diff
@@ -77,6 +77,12 @@ In a Georeference Annotations, these steps are stored as follows:
 | Transformation algorithm | A `transformation` property defined on the GeoJSON Feature Collection that holds the GCPs |
 {: .api-table #table-critical-data-for-georeferencing}
 
+#### 2.2.1 Handling Projections
+
+Please note that version 1.1 of this extension has no explicit provisions for indicating which projection, if any, was used during the georeferencing process described above.
```
Comment (glenrobson) [inline source/extension/georef/index.md]: Just to note we don't tend to version extensions but keep track of changes in the change log at the bottom of the document.

Comment (mradamcox) [reply source/extension/georef/index.md]: Oh sounds good, my misunderstanding. I'll update that wording.

**Comment by mradamcox** [comment]
@bertspaan converted to draft until I can take one more stab at rewording this.

**Comment by mradamcox** [comment]
Reworked the concept a bit here, after discussion with @sammeltassen (on Slack)

**Pull request merged**